src/mail.sh

```diff
@@ -46,10 +46,39 @@ function mail_setup()
   return 0
 }
 
+function mail_verify()
+{
+  local count=0
+  local -a missing_conf
+  local -a min_conf=('user.name' 'user.email' 'sendemail.smtpencryption'
+    'sendemail.smtpserver' 'sendemail.smtpuser' 'sendemail.smtpserverport')
```

Comment:
Keep in mind that git send-email is not part of the git package. You need to update the `setup.sh` to add git send email as an external dependency.